install/kubernetes/helm/istio/example-values/values-istio-meshexpansion-gateways.yaml

```diff
@@ -0,0 +1,29 @@
+global:
+  # Provides dns resolution for global services
+  podDNSSearchNamespaces:
+  - global
+  - "{{ valueOrDefault .DeploymentMeta.Namespace \"default\" }}.global"
```

Comment:
I am prtty sure ethis oine should as of now be purged throughout all example0-values yaml files. (line 5)

Reply:
Removed line 5.